server/src/handlers/chunk_handler.rs

```diff
@@ -47,8 +47,8 @@ pub struct CreateChunkData {
     pub chunk_vector: Option<Vec<f32>>,
     /// Tracking_id is a string which can be used to identify a chunk. This is useful for when you are coordinating with an external system and want to use the tracking_id to identify the chunk.
     pub tracking_id: Option<String>,
-    /// Group is the id of the group that the chunk should be placed into. This is useful for when you want to create a chunk and add it to a group in one request.
-    pub group_id: Option<uuid::Uuid>,
+    /// Group ids are the ids of the groups that the chunk should be placed into. This is useful for when you want to create a chunk and add it to a group or multiple groups in one request. Necessary because this route queues the chunk for ingestion and the chunk may not exist yet immediatley after response.
+    pub group_ids: Option<Vec<uuid::Uuid>>,
```

Comment:
Does utopia auto update?

Reply:
no, I wrote this

Reply:
No will utopia update the json parameters based on this

Reply:
Yeah it does